MainWindow.xaml.cs

```diff
@@ -231,7 +235,10 @@ private FlowDocument CreateFlowDocument(bool success, string serializedStyle)
                                     }
 
                                     paragraph.AddRun(_quotesStyle, "\"");
+                                    paragraph.AddLineBreak();
+                                    paragraph.AddRun(_textStyle, new string(' ', indent * 4 + elementName.Length + 1));
```

Comment:
I like to have each xml attribute on it's own line.
It greatly improves readabillity.